server/lib/amplitude.js

```diff
@@ -19,6 +19,7 @@ const logger = require('./logging/log')();
 const ua = require('./user-agent');
 
 const SERVICES = require('./configuration').get('oauth_client_id_map');
```

Comment:
`require('./configuration')` is called twice, before either of these lines can you set a variable `const config=require('./configuration');` and update these two lines to use the `config` reference?

Reply:
sure